dref/models.py

```diff
@@ -401,14 +407,16 @@ class Meta:
         unique_together = ('dref', 'country')
 
 
-class DrefImage(models.Model):
-    """
-    Multiple images are uploaded
-    """
-    dref = models.ForeignKey(Dref, verbose_name=_('dref'),
-                             on_delete=models.CASCADE)
-    image = models.FileField(
-        verbose_name=_('image'), null=True, blank=True,
-        upload_to=image_path, storage=get_storage(),
-        help_text=_('Images of the event')
+def dref_path(instance, filename):
+    return f'dref/{filename}'
```

Comment:
We don't need to define a function for simple use-cases.

https://docs.djangoproject.com/en/3.2/ref/models/fields/#django.db.models.FileField.upload_to